client/src/Components/header/Header.jsx

```diff
@@ -65,7 +65,7 @@ function Header() {
           {' '}
           <div className="shopping-cart-div">
             <ShoppingCart className="shopping-cart-icon" />
-            <p className="products-number">{cartitems.length}</p>
+            <p className="products-number">{0}</p>
```

Comment:
why you are doing this (static number) ?!

Reply:
react server didn't run when there  no item to the cart, I wait to merge the header component to the main